modules/apps/27-interchain-accounts/controller/ibc_middleware_test.go

```diff
@@ -367,7 +367,7 @@ func (suite *InterchainAccountsTestSuite) TestChanOpenConfirm() {
 
 	// use chainA (controller) for ChanOpenConfirm
 	msg := channeltypes.NewMsgChannelOpenConfirm(path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID, proofAck, proofHeight, icatypes.ModuleName)
-	handler := suite.chainA.GetSimApp().MsgServiceRouter().Handler(msg)
+	handler := suite.chainA.GetSimApp().BaseApp.MsgServiceRouter().Handler(msg)
```

Comment:
Post-sdk-middleware changes this seems to be the correct way to summon the MsgServiceRouter.